ios/FluentUI/Drawer/DrawerController.swift

```diff
@@ -370,6 +371,10 @@ open class DrawerController: UIViewController {
     }
     private var containerViewCenterObservation: NSKeyValueObservation?
 
+    /// for iPad split mode, navigation bar has a different dark elevated color, and if it is a `.down` presentation style, match `Colors.NavigationBar.background` elevated color
+    private var useNavigationBarBackgroundColor: Bool = false
+    private var useCustomBackgroundColor: Bool = false
```

Comment:
nit: do we still do alphabetical order?

Reply:
good point. I guess right now we do. I will move it around.